src/components/Menu/Menu.tsx

```diff
@@ -56,24 +68,33 @@ export const Menu = ({ className, ...other }: MenuProps) => {
 
 /**
  * A button that when clicked, shows or hides the Options
+ * Refer to Button documentation for more style variants.
  */
-const MenuButton = ({ children, className, ...other }: MenuButtonProps) => {
+const MenuButton = ({
+  children,
+  className,
+  showExpandIcon = true,
+  status,
+  ...other
+}: MenuButtonProps) => {
   const buttonClassNames = clsx(styles['menu__button'], className);
   return (
     <HeadlessMenu.Button
-      as={ClickableStyle}
+      as={Button}
       className={buttonClassNames}
-      status="neutral"
+      status={status || 'neutral'}
       {...other}
     >
       <>
         {children}
-        <Icon
-          className={styles['menu__button--icon']}
-          name="expand-more"
-          purpose="decorative"
-          size="1.25rem"
-        />
+        {showExpandIcon && (
+          <Icon
+            className={styles['menu__button--icon']}
```

Comment:
nit: the modifier bit should be verb-like, but `icon` by itself acts like an element. Icon is also generic (this particular one is always the chevron). `with-chevron` comes to mind, but something more like that would tell more about the intent

Reply:
I think `withChevron` sounds good